kubectl-rook-ceph.sh

```diff
@@ -339,12 +351,17 @@ function parse_main_flag () {
     "-n"|"--namespace")
       val_exists "$val" || return 1 # val should exist
       NAMESPACE="${val}"
+      CEPH_CONF_PATH="/var/lib/rook/$NAMESPACE/$NAMESPACE.config"
```

Comment:
I believe this isn't necessary if you move the other lines move as suggested above.